python/ray/autoscaler/v2/scheduler.py

```diff
@@ -189,7 +190,32 @@ def try_schedule(
                 - the utilization score for this node with respect to the current
                 resource requests being scheduled.
         """
-        pass
+        # Track the resource requests that cannot be scheduled on this node.
+        unschedulable_requests = []
+
+        def _sort_resource_request(req: ResourceRequest) -> Tuple:
+            """
+            Sort the resource requests by:
```

Comment:
add some comments why order in this way?

Reply:
also, should we sort the requests only once, instead of per node?

Reply:
good point - i think once is fine.